src/block/progress-circle/save.js

```diff
@@ -71,6 +72,7 @@ export const Save = props => {
 					aria-valuemax="100"
 					aria-valuenow={ progressValue }
 					aria-valuetext={ attributes.progressAriaValueText ? striptags( attributes.progressAriaValueText ) : undefined }
+					{ ...applyFilters( 'stackable.progress-circle.accessibility', attributes.progressAriaValueText, props.version ) }
```

Comment:
Instead of the filter adding the `aria-label`, the `aria-label` should be added normally, and then the filter would remove it for deprecation. this would also need to be done in the save portion

```suggestion
					aria-label={ attributes.progressAriaValueText ? striptags( attributes.progressAriaValueText ) : undefined }
					{ ...applyFilters( 'stackable.progress-circle.div-props', {}, props.version ) }
```

Then in the deprecation
```
const addAriaLabel = ( props, version ) => {
	// If from an old version, remove the aria-label
	if ( oldVersion ) {
		return { 
			...props,
			'aria-label': undefined,
		}
	}
	return props
}
addFilter( 'stackable.progress-circle.div-props', 'stackable/3.19.0', props, removeAriaLabel )
```